.circleci/config.yml

```diff
@@ -44,7 +44,10 @@ jobs:
             export SCENARIOO_DATA=./scenarioo-docu-generation-example/build/scenarioDocuExample
             export SPRING_SECURITY_USER_PASSWORD=justfortest
             echo "Starting Scenarioo"
-            java -Xms800m -Xmx800m -jar scenarioo-server/build/libs/scenarioo-latest.war > test-results/scenarioo-server.log || echo "Scenarioo server died. Exit code: $? see https://circleci.com/blog/how-to-handle-java-oom-errors/" >> test-results/scenarioo-server.log &
+            # To prevent a race condition between Spring Boot and ElasticSearch both trying to configure Netty
+            # we have to disable this through a System Property.
+            # See: https://discuss.elastic.co/t/contradictory-and-sometimes-poor-advice-given-for-es-set-netty-runtime-available-processors/148014
+            java -Xms800m -Xmx800m -Des.set.netty.runtime.available.processors=false -jar scenarioo-server/build/libs/scenarioo-latest.war > test-results/scenarioo-server.log || echo "Scenarioo server died. Exit code: $? see https://circleci.com/blog/how-to-handle-java-oom-errors/" >> test-results/scenarioo-server.log &
```

Comment:
Couldnt we set this property somehow differently - this is very ugly if each user has to set this

Reply:
So I would rather like to have a workaround in code that sets the property programatically.

Reply:
I will have another look. The first couple of things I tried with a programatic solution didn't work, because the methods where I implemented a change were triggered too late.

Reply:
Solution with @PostConstruct in ScenariooViewerApplication worked. Reverted change in config.yml